### ISSUE TITLE
Error: GITHUB_EVENT_NAME is not set when running octocov ls-files locally

### ISSUE THREAD

**Issue by shiroemons** [k1LoW/octocov#291]
I'm trying to use `octocov ls-files` in a local environment, but I encounter the following error message:

```bash
$ octocov --version
octocov version 0.54.3
$ octocov ls-files
Error: env GITHUB_EVENT_NAME is not set
```

Is the `GITHUB_EVENT_NAME` environment variable required for running `octocov` commands locally? If so, could you provide more information on why it's necessary? If not, could this be considered a bug?

Thank you for your help.

**Comment by k1LoW** [comment]
Probably because of using the `coverage.if:` section.

`octocov ls-files` should run locally as much as possible.

I'll fix it as soon as possible.

**Comment by shiroemons** [comment]
I'm using the `.octocov.yml` file generated by `octocov init`.
```yml
# generated by octocov init
coverage:
  if: true
codeToTestRatio:
  code:
    - '**/*.go'
    - '!**/*_test.go'
  test:
    - '**/*_test.go'
testExecutionTime:
  if: true
diff:
  datastores:
    - artifact://${GITHUB_REPOSITORY}
comment:
  if: is_pull_request
summary:
  if: true
report:
  if: is_default_branch
  datastores:
    - artifact://${GITHUB_REPOSITORY}

```

**Comment by k1LoW** [comment]
@shiroemons Thank you for your report!!

Released as [v0.54.4](https://github.com/k1LoW/octocov/releases/tag/v0.54.4).

**Comment by shiroemons** [comment]
@k1LoW Thank you for the quick response and fix!
